lib/bunch/sref.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Bunch.Sref do
+  @moduledoc """
+  A bunch of stuff for shortly-inspectable references.
```

Comment:
* one function is hardly a bunch
* this description could be used if such thing as 'shortly-inspectable references' existed. However, it's introducing that, so it should rather explain what it is